packages/components/src/mobile/inserter-button/index.native.js

```diff
@@ -80,13 +81,20 @@ class MenuItem extends Component {
 							isClipboardBlock && clipboardBlockStyles,
 						] }
 					>
-						<View style={ modalIconStyle }>
-							<Icon
-								icon={ item.icon.src || item.icon }
-								fill={ modalIconStyle.fill }
-								size={ modalIconStyle.width }
-							/>
-						</View>
+						<Badge
+							label={ __( 'New' ) }
```

Comment:
I'd read an [earlier comment](https://github.com/WordPress/gutenberg/pull/33119#issuecomment-887003936) about concerns that the badge would overlap the icon. Is this more of a concern when dealing with other languages where the translation for "New" could be a longer string?

I tested out Spanish by explicitly changing this line to "Nuevo" and the badge overlaps the icon significantly. A few alternatives came to mind which I tested out and added screenshots below. 

| Alternative | Screenshot |
| - | - |
| Current (in Spanish) | <img width="300" alt="Screen Shot 2021-07-27 at 19 38 29" src="https://user-images.githubusercontent.com/1898325/127241331-b9c72b14-74f2-4981-819b-0ad2e3b81f11.png"> |
| Emoji badge | <img width="300" alt="Screen Shot 2021-07-27 at 19 46 08" src="https://user-images.githubusercontent.com/1898325/127241946-d73e08e4-b7ef-432d-aa09-ca29b5f74ed0.png"> |
| Emoji in block title | <img width="300" alt="Screen Shot 2021-07-27 at 19 49 27" src="https://user-images.githubusercontent.com/1898325/127241961-44af6fc3-f195-4d78-bf1c-f477ac81680f.png"> |
| Emoji and text in block title (in Spanish) | <img width="300" alt="Screen Shot 2021-07-27 at 19 59 41" src="https://user-images.githubusercontent.com/1898325/127242558-ba4423c3-e4a1-4dc6-b11c-14cbbb91ff6b.png">

Reply:
@guarani this is a great point, and one that I had not considered. Another alternative — similar to the emoji — could be a simple "unread" indicator, much like iOS' Messages app.

![iOS Messages Unread Indicator](https://user-images.githubusercontent.com/438664/127244484-64bf7b76-3a7c-4ea2-89ea-c9a695acdbe3.jpeg)

@kyleaparker curious if you have thoughts on this issue regarding i18n and UI collisions.

Reply:
One related point is that the current new flag is not very visible in dark mode. I realize you probably just haven't gotten to the point of addressing this yet, but I wanted to raise it since I did notice it.

<img src="https://user-images.githubusercontent.com/4656348/127327157-e18de434-df70-4201-b7b3-d401f7314e8f.png" width="400px"/>

Reply:
> One related point is that the current new flag is not very visible in dark mode. 

Thanks @mchowning. I updated https://github.com/WordPress/gutenberg/pull/33119#issuecomment-887003936 to include screenshots of the updated dark mode styles. Awaiting @kyleaparker's feedback.

Reply:
Ah yes, sorry I missed that! Those do look much better! 😄

Reply:
Good point about other languages causing more problems. My preference would be to go with the star emoji badge on the top left corner. It's a little ambiguous but our main intention is to let people know about new badges and the star will draw people's attention. 

We have some upcoming design updates to the inserter that should give us more space to work with, but I think this is a good compromise for now.